hfs/btree.c

```diff
@@ -792,6 +791,7 @@ static int growBTree(BTree* tree) {
     
     while(TRUE) {
       descriptor = readBTNodeDescriptor(mapNode, tree);
+      ASSERT(descriptor, "readBTNodeDescriptor");
```

Comment:
It would be better to have a descriptive error message, such as "Failed to read BTNodeDescriptor".